code/modules/clothing/neck/_neck.dm

```diff
@@ -18,36 +18,119 @@
 			. += mutable_appearance('icons/effects/blood.dmi', "maskblood")
 
 /obj/item/clothing/neck/tie
-	name = "tie"
+	name = "clip-on tie"
 	desc = "A neosilk clip-on tie."
 	icon = 'icons/obj/clothing/neck.dmi'
-	icon_state = "bluetie"
+	icon_state = "detective"
 	inhand_icon_state = "" //no inhands
 	w_class = WEIGHT_CLASS_SMALL
 	custom_price = PAYCHECK_CREW
 
-/obj/item/clothing/neck/tie/blue
+/obj/item/clothing/neck/tie/greyscale
+	greyscale_colors = "#4d4e4e"
+	name = "slick tie"
+	desc = "A neosilk tie."
+	icon_state = "tie_greyscale_tied"
+	greyscale_config = /datum/greyscale_config/ties
+	greyscale_config_worn = /datum/greyscale_config/ties_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+	/// All ties start untied unless otherwise specified
+	var/is_tied = FALSE
+	/// How long it takes to tie the tie
+	var/tie_timer = 4 SECONDS
+
+/obj/item/clothing/neck/tie/greyscale/Initialize(mapload)
+	. = ..()
+	update_appearance(UPDATE_ICON)
+	if(usr)
+		if(ishuman(usr))
+			var/mob/living/carbon/human/H = usr
+			H.update_clothing(ITEM_SLOT_NECK)
+	register_context()
+
+/obj/item/clothing/neck/tie/greyscale/AltClick(mob/user)
+	. = ..()
+	to_chat(user, span_notice("You concentrate as you begin [is_tied ? "untying" : "tying"] [src]..."))
+	var/tie_timer_actual = tie_timer
+	// Mirrors give you a boost to your tying speed. I realize this stacks and I think that's hilarious.
+	for(var/obj/structure/mirror/reflection in view(2, user))
+		tie_timer_actual = tie_timer_actual / 1.25
+	// Heads of staff are experts at tying their ties.
+	if(user.mind.assigned_role.departments_bitflags & DEPARTMENT_BITFLAG_COMMAND)
+		tie_timer_actual = tie_timer_actual / 2
```

Comment:
```suggestion
		tie_timer_actual /= 2
```
el operator

Reply:
For some reason I didn't think the operator worked, oops